src/plugins/itemdisplay/MQ2ItemDisplay.cpp

```diff
@@ -1622,6 +1696,17 @@ class CItemDisplayWndOverride : public WindowOverride<CItemDisplayWndOverride, C
 		SetItem_Trampoline(pItem, flags);
 
 		s_inSetItem = false;
+
+		if (GetGameState() == GAMESTATE_INGAME && s_settings.PersistWindowBounds())
```

Comment:
game state will always be GAMESTATE_INGAME here.

Reply:
I thought so too, but I was getting random crashes when zoning, and these checks solved them.